tests/contrib/asyncpg/test_asyncpg.py

```diff
@@ -0,0 +1,154 @@
+from typing import Generator
+
+import asyncpg
+import pytest
+import pytest_asyncio
+
+from ddtrace import Pin
+from ddtrace.contrib.asyncpg import patch
+from ddtrace.contrib.asyncpg import unpatch
+from tests.contrib.config import POSTGRES_CONFIG
+
+
+@pytest.fixture(autouse=True)
+def patch_asyncpg():
+    # type: () -> Generator[None, None, None]
+    patch()
+    yield
+    unpatch()
+
+
+@pytest_asyncio.fixture
+async def patched_conn():
+    # type: () -> Generator[asyncpg.Connection, None, None]
+    conn = await asyncpg.connect(
+        host=POSTGRES_CONFIG["host"],
+        port=POSTGRES_CONFIG["port"],
+        user=POSTGRES_CONFIG["user"],
+        database=POSTGRES_CONFIG["dbname"],
+        password=POSTGRES_CONFIG["password"],
+    )
+    yield conn
+    await conn.close()
```

Comment:
try/finally?

Reply:
I'm pretty sure pytest fixtures are run in such a way that they'll always resume https://docs.pytest.org/en/6.2.x/fixture.html#handling-errors-for-yield-fixture